dev-python/sabyenc/sabyenc-5.4.2.ebuild

```
@@ -0,0 +1,24 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{8..11} )
+inherit distutils-r1
+
+DESCRIPTION="Module providing raw yEnc encoding/decoding for SABnzbd"
+HOMEPAGE="https://github.com/sabnzbd/sabyenc"
+SRC_URI="https://github.com/sabnzbd/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+BDEPEND="test? (
+		dev-python/chardet[${PYTHON_USEDEP}]
+	)"
```

This looks out of style for python packages :)
```suggestion
BDEPEND="
	test? (
		dev-python/chardet[${PYTHON_USEDEP}]
	)
"
```